bin/v-add-mail-domain-webmail

```diff
@@ -128,46 +128,46 @@ else
 		fi
 	fi
 
-	add_webmail_config "$WEB_SYSTEM" "${WEBMAIL_TEMPLATE}.tpl"
+	add_webmail_config ${WEB_SYSTEM} ${WEBMAIL_TEMPLATE}.tpl
 
 	if [ -n "$PROXY_SYSTEM" ]; then
-		add_webmail_config "$PROXY_SYSTEM" "${PROXY_TEMPLATE}.tpl"
+		add_webmail_config ${PROXY_SYSTEM} ${PROXY_TEMPLATE}.tpl
 	fi
 
 	# Enable SSL for webmail if available
-	if [ -f $HOMEDIR/$user/conf/mail/$domain/ssl/$domain.crt ] || [ "$SSL" = 'yes' ]; then
-		add_webmail_config "$WEB_SYSTEM" "${WEBMAIL_TEMPLATE}.stpl"
+	if [ -f ${HOMEDIR}/${user}/conf/mail/${domain}/ssl/${domain}.crt ] || [ "$SSL" = 'yes' ]; then
+		add_webmail_config ${WEB_SYSTEM} ${WEBMAIL_TEMPLATE}.stpl
 
 		if [ -n "$PROXY_SYSTEM" ]; then
-			add_webmail_config "$PROXY_SYSTEM" "${PROXY_TEMPLATE}.stpl"
+			add_webmail_config ${PROXY_SYSTEM} ${PROXY_TEMPLATE}.stpl
 		fi
 	fi
 fi
 
-WEBMAIL=$(get_object_value 'web' 'DOMAIN' "$domain" "$WEBMAIL")
+WEBMAIL=$(get_object_value 'web' 'DOMAIN' ${domain} ${WEBMAIL})
 if [ -z "$WEBMAIL" ]; then
-	add_object_key 'mail' 'DOMAIN' "$domain" 'WEBMAIL' 'SSL'
+	add_object_key 'mail' 'DOMAIN' ${domain} 'WEBMAIL' 'SSL'
 fi
 
 # Set SSL as enabled in configuration
-update_object_value 'mail' 'DOMAIN' "$domain" '$WEBMAIL' "$webmail"
+update_object_value 'mail' 'DOMAIN' ${domain} '$WEBMAIL' ${webmail}
```

Comment:
Also, removing quotes here is definitely unsafe coding practice, if some variable accidentally loses value, it will f**k everything up...